app/controllers/idv/capture_doc_status_controller.rb

```diff
@@ -90,5 +92,11 @@ def idv_session
         service_provider: current_sp,
       )
     end
+
+    def redo_document_capture_pending?
+      return unless session_result&.captured_at
+
+      document_capture_session.requested_at > session_result.captured_at
```

Comment:
yes i wanted to do that. however, this [line](https://github.com/18F/identity-idp/blob/36077adc82447a7c4a62d266126cd8069f6a067f/app/controllers/idv/capture_doc_status_controller.rb#L46) is the blocker ... do you knowi is this load_async still necessary? 🤔

Reply:
I think the `load_doc_auth_async_result` is related to the now-defunct async document capture code, yes. So it would probably be safe to remove all of that.

Reply:
thanks @aduth  ... after briefly testing out the removal of ```load_doc_auth_async_result```` .. i thnk it'd be best to remove it in a follow up PR